### PR TITLE
Better development environment.

### PR DESCRIPTION
**Pull request by RohanJnr**
- Environment variables for DEBUG, SUPERUSER_USERNAME, SUPERNAME_PASSWORD, SECRET_KEY, and DATABASE_URL.
- If DEBUG is True, SECRET_KEY is not required.
- Default value of  SUPERUSER_USERNAME, and SUPERNAME_PASSWORD is `admin`.
- Superuser is automatically created during the server start.
- Run the server with the help of pipenv to load the environment variables also.
- Update readme.

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -3,8 +3,27 @@
...
+
+def create_super_user() -> None:
+    """Create default super user for development mode."""
+    django.setup()
+    username = os.environ.get("SUPERUSER_USERNAME", "admin")
+    password = os.environ.get("SUPERUSER_PASSWORD", "admin")
+
+    user = get_user_model()
+
+    if not user.objects.filter(username=username).exists():
```
Comment (anihm136) [inline manage.py:19]: We also want to check if any superuser exists already (created manually, for example) and not create the default one if it does

**Review by anihm136 — CHANGES_REQUESTED** [review]

File: manage.py (hunk 1 above)
Comment (RohanJnr) [reply manage.py:19]: the superuser is being created only during development and it should not cause any problems as the user can specify superuser username and password as environment variables.

Comment (RohanJnr) [reply manage.py:19]: Also, do we want to create superuser automatically in production also?

Comment (anihm136) [reply manage.py:19]: Nope

**Pull request merged**